webapp/views.py

```diff
@@ -28,6 +29,10 @@
 from webapp.decorators import store_maintenance
 
 
+class NoAPIKeyError(Exception):
```

Comment:
why not:

```python3
from canonicalwebteam.search.views import NoAPIKeyError
```

?

Reply:
Cause I didn't think of it! Nice catch, I've changed it